src/components/TextInput/BaseTextInput/index.tsx

```diff
@@ -380,6 +380,7 @@ function BaseTextInput(
                                     // Add disabled color theme when field is not editable.
                                     inputProps.disabled && styles.textInputDisabled,
                                     styles.pointerEventsAuto,
+                                    isMarkdownEnabled ? {lineHeight: 18} : null,
```

Comment:
```suggestion
                                    isMarkdownEnabled ? {lineHeight: variables.lineHeightLarge} : null,
```

makes sense (or perhaps a different variable)?